### DISCUSSION TITLE
2.6.3 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2105] [讨论]
· 崩溃分析优化
　　支持分析 Mod 要求 Java 11 的崩溃（#2048，#2055，@SorrowRain，by @zkitefly）
　　优化 OptiFine 与 Forge 不兼容的崩溃检测（#2057，by @zkitefly）
　　扩大对 32 位 Java 分配内存过多导致崩溃的检测范围
　　扩大 Mixin 失败导致崩溃的检测范围
　　崩溃分析时，如果 PCL 不是最新版会提示更新
· 个别回声洞内容没有针对 UI 布局更新（#1918，@blibilijojo）
· 略微加粗了滑条以增加辨识度（#1951，@Sencheng06）
· 在尝试安装 Quilt 整合包时提示不支持（#1954，by @zkitefly）
· MC 更新提示的下载按钮将跳转到下载页面（#1991，@xtrackc1437）
· 资源下载的版本选项里添加 1.20.1（#2044，@WForst-Breeze）
· 新建 MC 文件夹时自动创建 launcher_profile.json（#2076，@hejiehao）
· 修改内存优化的提示，添加可能导致卡顿的描述
· 更新 5 月赞助者列表，感谢大家的支持！
· 修复了以下 14 个 Bug
　　使用部分中文输入法时输入离线用户名导致程序卡死（#1424，@Chen-Luan）
　　百宝箱按钮可能显示不全（#1915，@Mcenahle）
　　被意外最大化后无法还原窗口（#1938，@niguolaiap）
　　下拉列表项的提示位置在特定情况下居左（#1944，@amd64github）
　　可以在游戏启动时切换版本（#1969，@gyc123456-1）
　　特定情况下无法启动多个游戏（#2011，@KKW557，by @00ll00）
　　罕见情况下读取窗口大小失败导致程序崩溃（#2019，@KIJINSEIJA47）
　　无法正确处理已失效的指定的 Java（#2038，@Anythingteam）
　　为部分 Mod 错误地添加了“XX 版”提示（#2046，@WForst-Breeze）
　　崩溃分析时部分文件类型判断有误（#2052，@Sinecre88888）
　　1.12 OptiFine+Forge 的 Java 选择有误（#2052，@Sinecre88888）
　　错误地将官启自带 Java 视作优先指定的 Java（#2054，二次元文化）
　　主动取消删除版本导致报错（#2074，@misividkoukou）
　　主动取消删除 Mod 导致报错（#2082，@tchtcj）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.3'>2.6.3</a>.</em>

**Comment by iPodtouch6557** [comment]
真是的，直到29号我都在旅游，电脑没带（

**Reply by hejiehao** [reply]
你可以尝试一下云电脑

**Reply by iPodtouch6557** [reply]
懒（

**Comment by mcenahle** [comment]
👍👍👍
